xbmc/SortFileItem.h

```diff
@@ -19,51 +19,61 @@
+  SORT_METHOD_SIZE,                               //!< Integer: __4__ - Sort by File Size
+  SORT_METHOD_FILE,                               //!< Integer: __5__ - Sort by file name
+  SORT_METHOD_DRIVE_TYPE,                         //!< Integer: __6__ - Sort by drive type
+  SORT_METHOD_TRACKNUM,                           //!< Integer: __7__ - Sort by track number
+  SORT_METHOD_DURATION,                           //!< Integer: __8__ - Sort by duration
+  SORT_METHOD_TITLE,                              //!< Integer: __9__ - Sort by title
+  SORT_METHOD_TITLE_IGNORE_THE,                   //!< Integer: __10__ - Sort without the title
+  SORT_METHOD_ARTIST,                             //!< Integer: __11__ - Sort by Artist
+  SORT_METHOD_ARTIST_AND_YEAR,                    //!< Integer: __12__ - Sort by Artist and the year of them
+  SORT_METHOD_ARTIST_IGNORE_THE,                  //!< Integer: __13__ - Sort without Artist
+  SORT_METHOD_ALBUM,                              //!< Integer: __14__ - Sort by Album
+  SORT_METHOD_ALBUM_IGNORE_THE,                   //!< Integer: __15__ - Sort without album
+  SORT_METHOD_GENRE,                              //!< Integer: __16__ - Sort by Genre
+  SORT_METHOD_COUNTRY,                            //!< Integer: __17__ - Sort by the Country
+  SORT_METHOD_YEAR,                               //!< Integer: __18__ - Sort by the year
+  SORT_METHOD_VIDEO_RATING,                       //!< Integer: __19__ - Sort by video rating
+  SORT_METHOD_VIDEO_USER_RATING,                  //!< Integer: __20__ - Sort by video user rating
+  SORT_METHOD_DATEADDED,                          //!< Integer: __21__ - Sort by added date
+  SORT_METHOD_PROGRAM_COUNT,                      //!< Integer: __22__ - Sort by program count
+  SORT_METHOD_PLAYLIST_ORDER,                     //!< Integer: __23__ - Sort by playlist order
+  SORT_METHOD_EPISODE,                            //!< Integer: __24__ - Sort by episode number
+  SORT_METHOD_VIDEO_TITLE,                        //!< Integer: __25__ - Sort by video title
+  SORT_METHOD_VIDEO_SORT_TITLE,                   //!< Integer: __26__ - Sort by video sort title
+  SORT_METHOD_VIDEO_SORT_TITLE_IGNORE_THE,        //!< Integer: __27__ - Sort without video sort title
+  SORT_METHOD_PRODUCTIONCODE,                     //!< Integer: __28__ - Sort by song rating
+  SORT_METHOD_SONG_RATING,                        //!< Integer: __29__ - Sort by song user rating
+  SORT_METHOD_SONG_USER_RATING,                   //!< Integer: __30__ - Sort by user rating
+  SORT_METHOD_MPAA_RATING,                        //!< Integer: __31__ - Sort by MPAA rating
+  SORT_METHOD_VIDEO_RUNTIME,                      //!< Integer: __32__ - Sort by video runtime
+  SORT_METHOD_STUDIO,                             //!< Integer: __33__ - Sort by studio
+  SORT_METHOD_STUDIO_IGNORE_THE,                  //!< Integer: __34__ - Sort without the studio
```

Comment:
`Sort by studio, ignoring article (f.e. "Universal Pictures" _before_ "The Weinstein Company")`